data/test-segmentio.sh

```diff
@@ -2,21 +2,68 @@
 accessKey=$1
 
 # normal case
-curl -i -X POST http://localhost:7070/webhooks/segmentio.json?accessKey=$accessKey \
--H "Content-Type: application/json" \
--d '{
-  "version"   : 1,
-  "type"      : "identify",
-  "userId"    : "019mr8mf4r",
-  "traits"    : {
-      "email"            : "achilles@segment.com",
-      "name"             : "Achilles",
-      "subscriptionPlan" : "Premium",
-      "friendCount"      : 29
-  },
-  "timestamp" : "2012-12-02T00:30:08.276Z"
-}' \
--w %{time_total}
+curl -H "Accept: application/json; version=2.0" \
+     http://spec.segment.com/generate/identify | \
+curl -X POST \
+    -H "Content-Type: application/json" \
+    -d @- \
+    http://localhost:7070/webhooks/segmentio.json?accessKey=$accessKey
+
```

Comment:
I got this "name" missing error. is it expected that we should reject this data?

{"message":"Cannot convert JObject(List((channel,JString(server)), (context,JObject(List((campaign,JObject(List((content,JString(image-link)), (source,JString(Newsletter)), (medium,JString(email)), (term,JString(tps reports)), (name,JString(paldava))))), (device,JObject(List((manufacturer,JString(LGE)), (model,JString(LG-D620)), (type,JString(android)), (id,JString(60737ed9-2164-53c9-82ed-9f5f4d2336ed)), (name,JString(vagma)), (ad_tracking_enabled,JBool(false)), (advertising_id,JString(fizugol))))), (locale,JString(mo)), (network,JObject(List((bluetooth,JBool(true)), (carrier,JString(T Mobile)), (cellular,JBool(false)), (wifi,JBool(true))))), (page,JObject(List((referrer,JString(os))))), (screen,JObject(List((height,JInt(492602176569344)))))))), (integrations,JObject(List())), (message_id,JString(adf6dac2-8b9a-501c-916a-1f4520f7579f)), (project_id,JString(93afebb5)), (received_at,JString(2015-06-23T01:05:06.209Z)), (sent_at,JString(2015-06-23T01:04:06.209Z)), (timestamp,JString(2015-06-23T01:04:06.209Z)), (type,JString(page)), (user_id,JString(52373dfd)), (version,JString(2.0)), (properties,JObject(List((name,JString(cih)), (revenue,JInt(8364356471357440)), (value,JInt(-6538520982192128))))))) to event JSON. No usable value for name\nDid not find value which can be converted into java.lang.String"}